### ISSUE TITLE
 About training on other datasets/ Registration vs Fitting

### ISSUE THREAD

**Issue by boqian-li** [riccardomarin/NICP#10]
Hi @riccardomarin , there's another question about training on other datasets. 
I observed all the surface of output SMPL results are very close to the surface of input pointclouds or meshes. After checking the code of preprocessing CAPE dataset,  it comes to me that NICP cannot handle the shape difference between scan and inner SMPL when they are very different (**for example, a skinny person wearing very thick or loose clothes**, the output SMPL from NICP would be close to the surface of the outside surface of the cloth instead of the skinny body). 
In other words, this is a pure **registration** model instead of a **fitting** model. 

Above is the info I obtained after checking through the results code and paper, but I am not sure if it's true. Could you please provide more insights about that? :)

**Comment by riccardomarin** [comment]
Hi @boqian-li , 
Yes, the training, the NICP optimization, and the full NSR pipeline are designed to align the template with the target surface. Although possible, predicting a human shape under clothes is a different problem that would require quite some adaptations in the aforementioned components.

Best.

**Comment by boqian-li** [comment]
Thank you! I have better comprehension now!
